examples/js/loaders/GLTFLoader.js

```diff
@@ -1423,6 +1353,39 @@ THREE.GLTFLoader = ( function () {
 
 	};
 
+	/**
+	 * Requests all multiple dependencies of the specified types asynchronously, with caching.
+	 * @param {Array<string>} types
+	 * @return {Promise<Object<Array<Object>>>}
+	 */
+	GLTFParser.prototype.getMultiDependencies = function ( types ) {
+
+		var results = {};
+		var fns = [];
```

Comment:
maybe name this `deps` or `pending`; `fns` sounds like Functions, but these are Promises.